common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserSelector.java

```diff
@@ -233,7 +233,8 @@ private static boolean isCustomTabsServiceSupported(@NonNull final Context conte
 
     private static boolean isFullBrowser(final ResolveInfo resolveInfo) {
         // The filter must match ACTION_VIEW, CATEGORY_BROWSEABLE, and at least one scheme,
-        if (!resolveInfo.filter.hasAction(Intent.ACTION_VIEW)
+        if (resolveInfo.filter == null 
```

Comment:
Should we also check for `resolveInfo` itself being `null` as well? (before checking the resolve.filter)

Reply:
I don't think so. `resolveInfo` is already checked by the caller

Reply:
Ok, in that case let's add the `NonNull` annotation to the `resolveInfo` parameter to this method

Reply:
Added.